### ISSUE TITLE
Circuit Breaker: Document how to access Hystrix Dashboard.

### ISSUE THREAD

**Issue by cmoulliard** [openshiftio/appdev-documentation#300]
While the hystrix dashboard is optional, John is interested that we include it within the mission. Is it planned to include the instructions ?

**Comment by inoxx03** [comment]
Even though Hystrix Dashboard  is optional, I believe we should definitely include instructions for it. 
Personally, I'd be in favor of including the Hystrix Dashboard instructions as an appendix to the CB booster guide. WDYT?

**Comment by Ladicek** [comment]
Should this possibly be addressed in the booster implementations first?

**Comment by cmoulliard** [comment]
> appendix

I agree to keep the content under an appendix

**Comment by cmoulliard** [comment]
> booster implementations first?

Instructions should be similar between the missions. So why do you want to start with impl first ?

```
oc create -f https://raw.githubusercontent.com/snowdrop/openshift-templates/master/hystrix-dashboard/hystrix-dashboard.yml
oc new-app --template=hystrix-dashboard

# Get dashboard route to open it in our browser
oc get route/hystrix-dashboard --template='{{.spec.host}}'
hystrix-dashboard-cb.apps.35.187.106.198.nip.io

# Open into your browser
open this address http://hystrix-dashboard-cb.apps.35.187.106.198.nip.io in your browser to access the dashboard

# Replace the address of the application streaming the circuit data
Within the home page of the dashboard, replace the turbine address `http://turbine-server/turbine.stream` with the address of the `greeting service`, `http://springboot-cb-greeting/hystrix.stream`

```

**Comment by Ladicek** [comment]
If it's really this simple and doesn't require any changes in the boosters, then that's awesome :-) I simply didn't know and assumed that the boosters would have to be changed. Glad it isn't needed.

**Comment by cescoffier** [comment]
Depends on the runtime, for Vert.x we need to explicitly enable the hystrix event stream.

**Comment by cescoffier** [comment]
Once configured, it works.

![hystrix_monitor](https://user-images.githubusercontent.com/402301/27542552-da6c0976-5a87-11e7-928a-b0a870e1dc4e.png)

Data exposed on /metrics of the greeting service.

**Comment by inoxx03** [comment]
reference: http://vertx.io/docs/vertx-circuit-breaker/java/

**Comment by inoxx03** [comment]
Resolved in PR #329
